settings.yaml

```diff
@@ -78,7 +78,8 @@ nodestore:
   database: simple
 
 qdrant:
-  path: local_data/private_gpt/qdrant
+  url: ${PGPT_QDRANT_URL:localhost:6333}
+  hybrid_search: ${PGPT_QDRANT_HYBRID_SEARCH:true}
```

Comment:
Let's do this in a different PR, given it is not directly related. Also, II don't think we are using PGPT_* env vars anywhere else. I'd say let's keep it simple and avoid env vars. Users can just override using their own settings_x.yaml